llvm/lib/Transforms/Utils/Debugify.cpp

```diff
@@ -57,6 +65,50 @@ cl::opt<Level> DebugifyLevel(
 
 raw_ostream &dbg() { return Quiet ? nulls() : errs(); }
 
+#ifdef LLVM_ENABLE_DEBUGLOC_COVERAGE_TRACKING
+// These maps refer to addresses in this instance of LLVM, so we can reuse them
+// everywhere - therefore, we store them at file scope.
+static DenseMap<void *, std::string> SymbolizedAddrs;
+static DenseSet<void *> UnsymbolizedAddrs;
```

Comment:
Totally appropriate in the context of debugify, which is fundamentally a testing system. (Figured I'd chime in with the endorsement).